Orange/widgets/model/tests/test_owneuralnetwork.py

```diff
@@ -21,3 +21,14 @@ def test_migrate_setting(self):
         settings = dict(alpha=103)
         OWNNLearner.migrate_settings(settings, None)
         self.assertEqual(OWNNLearner.alphas[settings["alpha_index"]], 100)
+
+    def test_no_layer_warning(self):
+        self.widget.apply_button.button.click()
+
+        self.widget.hidden_layers_input = ""
+        self.widget.apply_button.button.click()
+        self.assertTrue(self.widget.Warning.no_layers.is_shown())
+
+        self.widget.hidden_layers_input = "10,"
+        self.widget.apply_button.button.click()
+        self.widget.apply_button.button.click()
```

Comment:
Did you want to check the widget's resilience to double clicks or did you mean `self.assertFalse(self.widget.Warning.no_layers.is_shown())`? :)

Reply:
Definitely wanted to make sure the button is certainly clicked. 😆 

Will fix it.